.github/workflows/execution-plan.yml

```diff
@@ -29,5 +29,94 @@ jobs:
           cat artifacts/execution-plan.json
       - uses: actions/upload-artifact@v3
         with:
-          name: execution-plan
-          path: artifacts/*
+          name: execution-plan.json
+          path: artifacts/execution-plan.json
+
+  github_convertor:
+    needs: execution_plan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@master
+        with:
+          name: execution-plan.json 
+          path: artifacts
+      - uses: google-github-actions/auth@v0 
+        name: 'Authenticate to Google Cloud'
+        with:
+          credentials_json: ${{ secrets.FDC_CORE_CI_IMAGE_READER }} 
+      - name: gcloud authorize
+        run: |
+          gcloud auth configure-docker
+      - name: Planner
+        id: plan
+        run: |
+          init="$(cat artifacts/execution-plan.json | docker run -i eu.gcr.io/freiheit-core/services/github-convertor/github-convertor:0.0.2 init | tr -d '\n')"
+          echo "::set-output name=init::${init}"
+          build="$(cat artifacts/execution-plan.json | docker run -i eu.gcr.io/freiheit-core/services/github-convertor/github-convertor:0.0.2 build | tr -d '\n')"
```

Comment:
why do you need to delete the newline in the end? Is that really necessary?

Reply:
output of github-convertor is indented, this is to convert it to one line json so that set-output works properly (set output doesn't work with multiline strings out of the box